### PR TITLE
Experiment JVM SDK

### PR DESCRIPTION
**Pull request by bgiori**
Docs for Experiment JVM SDK with Local Evaluation support.

### CODE REVIEW

File: docs/experiment/sdks/jvm-sdk.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+
+Official documentation for Amplitude Experiment's server-side JVM SDK implementation. This SDK may be used in **either Java or Kotlin** server-side implementations.
+
+!!!info "SDK Resources"
+    [:material-github: Github](https://github.com/amplitude/experiment-jvm-server) · [:material-code-tags-check: Releases](https://github.com/amplitude/experiment-jvm-server/releases)
+
+This documentation is split into two sections for [remote](../general/evaluation/remote-evaluation.md) and [local](../general/evaluation/local-evaluation.md) evaluation:
+
+* [Remote evaluation](#remote-evaluation)
+* [Local evaluation (alpha)](#local-evaluation-alpha)
```
Comment (qingzhuozhen) [inline docs/experiment/sdks/jvm-sdk.md:15]: Do we need to add cookie session for remote?

File: docs/experiment/sdks/jvm-sdk.md [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+
+The SDK client can be configured on initialization.
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div> | Description | Default Value |
+    | --- | --- | --- |
+    | `debug` | Set to `true` to enable debug logging. | `false` |
+    | `serverUrl` | The host to fetch flag configurations from. | `https://api.lab.amplitude.com` |
+    | `fetchTimeoutMillis` |  The timeout for fetching variants in milliseconds. This timeout only applies to the initial request, not subsequent retries | `500` |
+    | `fetchRetries` | The number of retries to attempt if a request to fetch variants fails. | `1` |
```
Comment (qingzhuozhen) [inline docs/experiment/sdks/jvm-sdk.md:152]: default retry once?

File: docs/experiment/sdks/jvm-sdk.md (hunk 1 above)
Comment (bgiori) [reply docs/experiment/sdks/jvm-sdk.md:15]: Yeah, I'm going to add that after the initial release. Good callout though.

File: docs/experiment/sdks/jvm-sdk.md (hunk 2 above)
Comment (bgiori) [reply docs/experiment/sdks/jvm-sdk.md:153]: Yeah, I think this is a better default. This is what we've landed on using in thunder for example.

**Review by qingzhuozhen — APPROVED** [review]

**Review by caseyamp — APPROVED** [review]

**Pull request merged**